app/controllers/api_controller.rb

```diff
@@ -203,11 +212,12 @@ def supported_relationships
 
   def api_version
     res = request.headers['Accept'].match(REGEXP_API_VERSION)
-    res&.[](:version) || '1'
+    res&.[](:version)
   end
 
   def extend_versioned_controller_actions
-    version = "Api::V#{api_version}"
+    default_version = '1'
```

Comment:
We could make this a constant rather than loading it on every request?

Reply:
I agree